www/app/Console/Commands/CleanupStaleConversations.php

```
@@ -41,28 +46,67 @@ public function handle(StreamManager $streamManager): int
 
         foreach ($staleConversations as $conversation) {
             try {
-                // Double-check: is the stream actually still active in Redis?
-                if ($streamManager->isStreaming($conversation->uuid)) {
-                    $this->warn("Skipping {$conversation->uuid}: stream still active in Redis");
-                    continue;
+                $redisStatus = $streamManager->getStatus($conversation->uuid);
+                $hasAbortFlag = $streamManager->checkAbortFlag($conversation->uuid);
+                $lastActivity = $conversation->last_activity_at;
+                $isOrphaned = false;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$isOrphaned'. (undefined)

(UnusedLocalVariable)